js/id/ui/raw_tag_editor.js

```diff
@@ -186,7 +186,7 @@ iD.ui.RawTagEditor = function(context) {
             .on('change', keyChange);
 
         $items.select('input.value')
-            .value(function(d) { return d.value; })
+            .attr('value', function(d) { return d.value; })
```

Comment:
It's similar, but adding the value as an attribute allowed cucumber to identify the element in a way that wasn't working using .value. It shows up as an attribute for the element as seen in this screen grab:
![screenshot from 2016-09-27 10 37 59](https://cloud.githubusercontent.com/assets/17727812/18878473/b02c8934-849e-11e6-99cb-854a6aabdffb.png)